### ISSUE TITLE
Start Page: Install commands updated for each version

### ISSUE THREAD

**Issue by bradygm** [pypose/pypose.github.io#32]
@wang-chen Should it just be pip install for all configurations? Anything specific for the two CUDA versions?

**Comment by wang-chen** [comment]
yes, it is,`pip install pypose` for all configurations. For different CUDA version, the users need to follow the instructions of PyTorch.

**Comment by wang-chen** [comment]
we currently only support PyTorch `1.12.1`
